CairoMakie/src/primitives.jl

```diff
@@ -453,6 +449,25 @@ function draw_glyph_collection(scene, ctx, position, glyph_collection, rotation,
     return
 end
 
+struct CairoGlyph
+    index::Culong
+    x::Cdouble
+    y::Cdouble
+end
+
+function show_glyph(ctx, glyph::Culong, x, y)
+    cg = Ref(CairoGlyph(glyph, x, y))
+    ccall((:cairo_show_glyphs, Cairo.libcairo),
+            Nothing, (Ptr{Nothing}, Ptr{CairoGlyph}, Cint),
+            ctx.ptr, cg, 1)
```

Comment:
In this case you should use [cairo_show_text_glyphs](https://www.cairographics.org/manual/cairo-text.html#cairo-show-text-glyphs), which allow text to be highlighted.  We had glyph support earlier but AFAIK this is why we switched away from it.

Reply:
At this point I don't have the utf 8 anymore and it's not guaranteed that there is any, as for Mathtexengine